test/Grpc.IntegrationTests/GrpcDurableTaskClientIntegrationTests.cs

```diff
@@ -150,6 +150,80 @@ await server.Client.ScheduleNewOrchestrationInstanceAsync(
         page.ContinuationToken.Should().NotBeNull();
     }
 
+    [Fact]
+    public async Task PurgeInstance_EndToEnd()
+    {
+        // Arrange
+        await using HostTestLifetime server = await this.StartAsync();
+
+        // Create and complete an orchestration instance
+        string instanceId = await server.Client.ScheduleNewOrchestrationInstanceAsync(
+            OrchestrationName, input: false);
+
+        // Wait for it to start and raise event to complete it
+        await server.Client.WaitForInstanceStartAsync(instanceId, default);
+        await server.Client.RaiseEventAsync(instanceId, "event", default);
+        await server.Client.WaitForInstanceCompletionAsync(instanceId, default);
+
+        // Verify instance exists before purge
+        OrchestrationMetadata? metadata = await server.Client.GetInstanceAsync(instanceId, false);
+        metadata.Should().NotBeNull();
+        metadata!.InstanceId.Should().Be(instanceId);
+        metadata.RuntimeStatus.Should().Be(OrchestrationRuntimeStatus.Completed);
+
+        // Act
+        PurgeResult result = await server.Client.PurgeInstanceAsync(
+            instanceId,
+            new PurgeInstanceOptions { Recursive = true });
+
+        // Assert
+        result.Should().NotBeNull();
+        result.PurgedInstanceCount.Should().Be(1);
+        result.IsComplete.Should().BeNull();
```

Comment:
Same comment that this should be either null or true, so maybe we should assert that it's not false.

Reply:
Same comment for the other use in the next test method.

Reply:
updated